manim/mobject/svg/tex_mobject.py

```diff
@@ -621,6 +621,21 @@ def fade_all_but(self, index_or_string, opacity=0.5):
 
 
 class Title(Tex):
+    """
+    Example
+    --------
+    .. manim:: TitleExample
+        :save_last_frame:
+
+        import manim
+        class TitleExample(Scene):
+            def construct(self):
+                banner = ManimBanner()
+                self.play(banner.expand())
+                ti=Title(f"manim version {manim.__version__}")
```

Comment:
I would prefer if variable names in examples were more verbose. I don't really see an issue with naming this mobject `title`, or maybe even `title_mobject`? Or `version_text`?

Examples should be easily read- and understandable; variable naming is an important part of that.